Crashlytics/Crashlytics/Components/FIRCLSUserLogging.m

```diff
@@ -32,6 +32,10 @@
 NSString *const FIRCLSDevelopmentPlatformNameKey = @"com.crashlytics.development-platform-name";
 NSString *const FIRCLSDevelopmentPlatformVersionKey =
     @"com.crashlytics.development-platform-version";
+NSString *const FIRCLSFlutterOnDemandRecordedExceptionsKey =
+    @"com.crashlytics.on-demand-recorded-flutter-exceptions";
```

Comment:
Can we rename this to just be on-demand, not flutter specific? I hope we use this for other platforms in the future.